<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RC NotePad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/Icon-notepad.png</normaloff>:/img/Icon-notepad.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>144</y>
      <width>271</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionNew_Window"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
    <addaction name="actionUpdate_Settings"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionNew_Window"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionColor"/>
   <addaction name="actionUpdate_Settings"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file-regular.png</normaloff>:/img/file-regular.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/folder-open-regular.png</normaloff>:/img/folder-open-regular.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/save-regular.png</normaloff>:/img/save-regular.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file-export-solid.png</normaloff>:/img/file-export-solid.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/copy-regular.png</normaloff>:/img/copy-regular.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/cut-solid.png</normaloff>:/img/cut-solid.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/clipboard-regular.png</normaloff>:/img/clipboard-regular.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/undo-solid.png</normaloff>:/img/undo-solid.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/redo-solid.png</normaloff>:/img/redo-solid.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/file-medical-solid.png</normaloff>:/img/file-medical-solid.png</iconset>
   </property>
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/book-dead-solid.png</normaloff>:/img/book-dead-solid.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/font-solid.png</normaloff>:/img/font-solid.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/color-icon-25.jpg</normaloff>:/img/color-icon-25.jpg</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionUpdate_Settings">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/1601833.png</normaloff>:/img/1601833.png</iconset>
   </property>
   <property name="text">
    <string>Update Settings</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/img/help_6.png</normaloff>:/img/help_6.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
